packages/components/src/styles/table-cell/table-cell.css

```diff
@@ -5,6 +5,32 @@
     @apply border-r-[1px];
   }
 
+  &--shadow-bottom {
+    &:after {
+      content: '';
+      position: absolute;
+      left: 0;
+      right: 0;
+      bottom: -10px;
+      height: 10px;
+      background: linear-gradient(to bottom, rgba(24, 24, 24, 0.1), transparent);
+      transition: opacity 10s linear;
```

Comment:
10 seconds? 😆 

And it feels that you could abstract some stuff.

&--shadow-bottom, &--shadow-right{

content: '';
transition...;
position: absolute;
}

...

So that we clearly see differnce.

I'm a bit irritated, that we need first-child for shadow-right? This should be part of the element directly?

The transition currenlty doesn't work. I believe you have to make the content and everythign visible and just change opacity etc.

Plus make sure to remove pointer-events, so no its clickthroughable.

Reply:
Oh and please use Tailwind here:

@left-0 right-0 etc.

Reply:
outdated